src/components/supportus/PhaseBudget.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useState, useEffect } from "react";
+import ideaImg from "@/assets/supportus/ideablue.jpg";
+import Image, { StaticImageData } from "next/image";
+
+type Phase = {
+  image: StaticImageData;
+  header: string;
+  text: string;
+};
+
+const phaseData: Phase[] = [
+  {
+    image: ideaImg,
+    header: "Phase 1: Ideation",
+    text: "Lorem Ipsum is simply dummy text of the printing and  typesetting industry. Lorem Ipsum has been the industry's standard  dummy text ever since the 1500s, when an unknown printer took a galley  of type and scrambled it to make a type specimen book. ",
+  },
+  {
+    image: ideaImg,
+    header: "Phase 2: Prototype",
+    text: "Lorem Ipsum is simply dummy text of the printing and  typesetting industry. Lorem Ipsum has been the industry's standard  dummy text ever since the 1500s, when an unknown printer took a galley  of type and scrambled it to make a type specimen book.",
+  },
+  {
+    image: ideaImg,
+    header: "Phase 3: Experiment",
+    text: "Lorem Ipsum is simply dummy text of the printing and  typesetting industry. Lorem Ipsum has been the industry's standard  dummy text ever since the 1500s, when an unknown printer took a galley  of type and scrambled it to make a type specimen book.",
+  },
+  {
+    image: ideaImg,
+    header: "Phase 4: Validation",
+    text: "Lorem Ipsum is simply dummy text of the printing and  typesetting industry. Lorem Ipsum has been the industry's standard  dummy text ever since the 1500s, when an unknown printer took a galley  of type and scrambled it to make a type specimen book.",
+  },
+];
+
+const PhaseBudget: React.FC = () => {
```

Comment:
Nothing big, just for consistency (and preferences)
```suggestion
function PhaseBudget() {
```

Reply:
The figma didnt have a button for that on desktop, would be easy to add but might clutter desktop view